build.sbt

```diff
@@ -282,8 +321,6 @@ lazy val readme = scalatex.ScalatexReadme(
 
 lazy val sharedSettings = Def.settings(
   scalaVersion := ScalaVersion,
-  crossScalaVersions := ScalaVersions,
-  crossVersion := CrossVersion.binary,
```

Comment:
Good idea, I started this change before I realised it was only two setting keys.